man/ostree.repo-config.xml

```diff
@@ -416,6 +416,14 @@ License along with this library. If not, see <https://www.gnu.org/licenses/>.
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>boot_counting</varname></term>
```

Comment:
Bikeshed: for consistency with other knobs in this file, `boot-counting`?

Also, should this be e.g. `boot-counting-tries` to make it clearer?

Reply:
Changed to `boot-counting-tries`